zubhub_frontend/zubhub/package.json

```diff
@@ -26,14 +26,15 @@
     "react-quill": "^1.3.5",
     "react-redux": "^7.2.2",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^4.0.3",
     "react-slick": "^0.27.13",
     "react-toastify": "^6.1.0",
     "redux": "^4.0.5",
     "redux-persist": "^6.0.0",
     "redux-thunk": "^2.3.0",
     "slick-carousel": "^1.8.1",
     "source-map-explorer": "^2.5.2",
+    "util": "^0.12.4",
```

Comment:
when i first ran npm install it did some adjustments and the util was part of that.. so it started causing errors because the util was in use but the dependency was no longer there... so i reverted back to the default packages but it still didn't get the util but the errors reduced to just the util so i added it and all the errors cleared cos it could now see the module. anyone running frontend on npm without that would be getting lots of errors and reload issues if their npm messes with that..

Reply:
I tried to install without docker and I see the package-lock.json was created with old npm and there are some changes in the file. We can create separate issue for updating NodeJS version (in docker) and prerequisites and also update package-lock.json

Reply:
@tuxology